src/definitions/modules/dropdown.js

```diff
@@ -1602,7 +1614,7 @@
                                 }
 
                                 // up arrow (traverse menu up)
-                                if (pressedKey == keys.upArrow) {
+                                if (pressedKey === keys.upArrow) {
```

Comment:
above, such if structure was converted into a switch/case construct. Why not here?

Reply:
because eslint rule did not fix it :)

I am not huge fan of switch/case constructs because it adds 1 indentation level and `break`s, so I do not want to pursue this to be fixed if not needed